validateFileNames.js

```diff
@@ -0,0 +1,94 @@
+const shelljs = require('shelljs');
+
+// Regex that defines our file and directory naming convention
+const camelCasedFileNameRegex = /^[.]?([a-z])+([0-9]|[a-zA-Z]|[.])*$/;
+
+// Files and directories that are exempt from the naming convention
+const ignoredFilesAndDirectories = [
+  'README.md',
+  'Dockerfile',
+  'LICENSE.md',
+  'customTypings',
+  'typings',
+];
```

Comment:
No, it's fine. This is a fun, easy one, and it's not urgent. I can handle it. 🙂

So, I'm thinking of using `process.cwd()` to look for the config file instead of `find-package-json` for simplicity's sake. Thoughts?